build.rs

```diff
@@ -81,5 +82,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         println!("cargo:rerun-if-changed={}", path.to_str().unwrap());
     }
 
+    built::write_built_file().expect("Failed to acquire build-time information");
```

Comment:
It's being used here to print out the commit hash when we start running. This is very useful for development because when we were deploying new iterations of this PR we weren't also incrementing the version number, so this lets easily verify that the correct code was deployed by looking at the logs. https://github.com/googleforgames/quilkin/blob/502946d6e4666b73a549166e9894c5a20e236ea5/src/main.rs#L125

Reply:
That's super good to have 👍🏻 

Ah, I got it - took me a bit to find the docs: https://docs.rs/built/latest/built/

100% makes sense now - thanks!!!